templates/DDL/REVOKE.j2

```diff
@@ -0,0 +1,23 @@
+{% for grant in actual_privs %}
+{% for priv in grant.access_type %}
+{% if (grant.user_name in item.to) or (grant.role_name in item.to)
```

Comment:
As this if has a lot of conditions, I would rather use `{% set %}` to give it a proper meaning